sci-electronics/systemc/systemc-2.3.4-r1.ebuild

```diff
@@ -41,12 +40,28 @@ src_configure() {
 		--with-unix-layout
 }
 
+src_install() {
+	default
+
+	if use doc; then
+		if use examples; then
+			docompress -x /usr/share/doc/"${PF}"/examples
```

Comment:
Because there are many examples file, and should keep decompressed, so I use ``docompress -x`` here to keep it decompressed.
I've test the ``doc`` and ``example`` use flag in my overlay and try to make it work correctly:
https://github.com/vowstar/vowstar-overlay/commit/ad2b499dbbf0482c79bce0a1fda379af38db72f5
https://github.com/vowstar/vowstar-overlay/commit/922ec8af0bfece141d8b727949d3bc73bafede53
https://github.com/vowstar/vowstar-overlay/commit/922ec8af0bfece141d8b727949d3bc73bafede53
https://github.com/vowstar/vowstar-overlay/commit/02a5240f9d3cb3382628f72cc5624a2b2ded399b
https://github.com/vowstar/vowstar-overlay/commit/48422bdf6ca66568164fa9e1d86f5d63659aef29
Finally it works.